mobile-app/app/screens/AppNavigator/screens/Dex/DexConfirmRemoveLiquidity.tsx

```diff
@@ -216,6 +216,11 @@ async function constructSignedRemoveLiqAndSend (pair: PoolPairData, amount: BigN
       symbol: symbol,
       amount: amount.toFixed(8)
     }),
+    drawerMessages: {
+      preparing: translate('screens/OceanInterface', 'Preparing to remove liquidity…'),
+      waiting: translate('screens/OceanInterface', 'Removing tokens from liquidity pool…'),
```

Comment:
Is this UI also part of this task?
![image](https://user-images.githubusercontent.com/13292662/183341071-86407f99-689f-4a33-b47f-992c221e0996.png)

Reply:
Same with add liquidity

Reply:
Yes, we updates all txn and added custom messages for preparing, waiting and completed status